config/purposes/heron_384.yml

```diff
@@ -0,0 +1,57 @@
+---
+LHR-384 RT:
+  :asset_type: plate
+  :stock_plate: true
+  :cherrypickable_target: true
+  :input_plate: true
+  :default_printer_type: :plate_a
+  :presenter_class: Presenters::StockPlatePresenter
+  :label_template: plate_6mm_double
+  :size: 384
+LHR-384 PCR 1:
+  :asset_type: plate
+  :label_template: plate_6mm_double
+  :size: 384
+LHR-384 PCR 2:
+  :asset_type: plate
+  :label_template: plate_6mm_double
+  :size: 384
+LHR-384 cDNA:
+  :asset_type: plate
+  :creator_class: LabwareCreators::MergedPlate
+  :merged_plate:
+    source_purposes:
+    - 'LHR-384 PCR 1'
+    - 'LHR-384 PCR 2'
+    help_text: 'Here we are merging the two Primer Panel PCR plates, creating a new cDNA plate.'
+  :label_template: plate_6mm_double
+  :size: 384
+LHR-384 XP:
+  :asset_type: plate
+  :merger_plate: true
+  :label_template: plate_6mm_double
+  :size: 384
+LHR-384 End Prep:
+  :asset_type: plate
+  :presenter_class: Presenters::PermissivePresenter
+  :label_template: plate_6mm_double
+  :size: 384
+LHR-384 AL Lib:
+  :asset_type: plate
+  :label_template: plate_6mm_double
+  :size: 384
+LHR-384 Lib PCR:
+  :asset_type: plate
+  :presenter_class: Presenters::MinimalPcrPlatePresenter
```

Comment:
Was this 'MinimalPcrPlatePresenter' the one that doesn't show the preview of the plate? Is that ok?